delegate-cli/src/cli/mod.rs

```diff
@@ -0,0 +1,35 @@
+use clap::{command, Parser, Subcommand};
+use seda_config::{DelegateConfig, PartialDelegateConfig};
+mod commands;
+
+mod utils;
+
+#[derive(Parser)]
+#[command(name = "seda-delegate")]
+#[command(author = "https://github.com/SedaProtocol")]
+#[command(version)]
+#[command(propagate_version = true)]
+#[command(about = "For staking & delegation on the SEDA protocol.", long_about = None)]
+#[command(next_line_help = true)]
+pub struct CliOptions {
+    #[command(flatten)]
+    pub delegate_config: PartialDelegateConfig,
+    #[command(subcommand)]
+    pub command:         Command,
+}
+
+#[derive(Debug, Subcommand)]
+pub enum Command {
+    TopUp(commands::top_up::TopUp),
```

Comment:
Add some comment to have the CLI `--help` show some command info.

```suggestion
    /// Top-up a target account by a given amount in NEAR
    TopUp(commands::top_up::TopUp),
```